topics/dev/tutorials/community-tool-table/tutorial.md

```diff
@@ -129,15 +130,13 @@ an example of the file that is used to manually filter the tools for a community
 >
 > 1. Download the `tools.tsv` file in `results/<your community>`.
 > 2. Open `tools.tsv` with a Spreadsheet Software.
-> 3. Review each line corresponding to a tool
-       
-       You can also just review some tools. Those tools that are not reviewed will be have `FALSE` in the `Reviewed` columns the updated table.
+> 3. Review each line corresponding to a tool. You can also just review some tools. Those tools that 
+> are not reviewed will have be set to `FALSE` in the `Reviewed` column of the updated table.
 >    1. Change the value in the `Reviewed` column from `FALSE` to `TRUE` (this will be done automatically if an entry of the tool in `tools_status.tsv` exists).
 >    2. Add `TRUE` to the `To keep` column if the tool should be kept, and `FALSE` if not.
 >    3. Add `TRUE` or `FALSE` also to the `Deprecated` column.
-> 4. Copy paste the `Galaxy wrapper id`, `To keep`, `Deprecated` column in a new table (in that order).
-
-       This can also be done using the reference function of your Spreadsheet Software.
+> 4. Copy paste the `Galaxy wrapper id`, `To keep`, `Deprecated` columns in a new table (in that order). This 
+> can also be done using the reference function of your Spreadsheet Software.
```

Comment:
```suggestion
> 4. Copy paste the `Galaxy wrapper id`, `To keep`, `Deprecated` columns in a new table (in that order). 
>
>      This can also be done using the reference function of your Spreadsheet Software.
```